src/Common/qMR_reportBug.m

```diff
@@ -0,0 +1,64 @@
+function qMR_reportBug(exception)
+% report bug to the qMRLab dev team.
```

Comment:
i would specify in the docstring that this function opens an issue on Github website